awxkit/awxkit/cli/custom.py

```diff
@@ -56,6 +56,9 @@ def add_arguments(self, parser, resource_options_parser, with_pk=True):
         parser.choices[self.action].add_argument('--monitor', action='store_true', help='If set, prints stdout of the launched job until it finishes.')
         parser.choices[self.action].add_argument('--action-timeout', type=int, help='If set with --monitor or --wait, time out waiting on job completion.')
         parser.choices[self.action].add_argument('--wait', action='store_true', help='If set, waits until the launched job finishes.')
+        parser.choices[self.action].add_argument(
+            '--interval', type=float, help='If set with --monitor or --wait, amount of time to wait in seconds between api calls.'
```

Comment:
Do you really want a float instead of an int here?

Reply:
With the new update and the minimum value set to 2.5, I think it would be a good idea to make the argument `int` (and minimum an integer larger than 0). Hopefully, nobody needs faster updates - it would make printing output faster/smoother but potentially overwhelming the server again.